### PR TITLE
Doc updates for JSON array indexing

### PR DESCRIPTION
**Pull request by slorello89**
Noticed that there wasn't a solid explanation or example for indexing arrays with tags in the docs, added essentially the same example in the tags doc and in the indexing with json doc

### CODE REVIEW

File: docs/Indexing_JSON.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ FT.SEARCH userIdx '@name:(John)'
 
 ## Indexing JSON arrays with tags
 
-It is possible to index scalar values in JSON arrays by using the wildcard operator in the JSON Path. For example if you were indexing blog posts you might have a field called `tags` which is an array of tags that apply to the blog post.
+It is possible to index scalar string values in JSON arrays by using the wildcard operator in the JSON Path. For example if you were indexing blog posts you might have a field called `tags` which is an array of tags that apply to the blog post.
```
Comment (oshadmi) [inline docs/Indexing_JSON.md:75]: Also boolean

File: docs/Tags.md [hunk 2]
```diff
@@ -116,7 +116,7 @@ FT.SEARCH punctuation "@tags:{ Andrew\\'s Top 5 }"
 
 ## Indexing JSON arrays with tags
 
-It is possible to index scalar values in JSON arrays by using the wildcard operator in the JSON Path. For example if you were indexing blog posts you might have a field called `tags` which is an array of tags that apply to the blog post.
+It is possible to index scalar string values in JSON arrays by using the wildcard operator in the JSON Path. For example if you were indexing blog posts you might have a field called `tags` which is an array of tags that apply to the blog post.
```
Comment (oshadmi) [inline docs/Tags.md:119]: Also boolean

**Review by oshadmi — COMMENTED** [review]
Looks great! Minor comment

File: docs/Indexing_JSON.md [hunk 3]
```diff
@@ -70,6 +70,39 @@ FT.SEARCH userIdx '@name:(John)'
    2) "{\"user\":{\"name\":\"John Smith\",\"tag\":\"foo,bar\",\"hp\":1000,\"dmg\":150}}"
 ```
 
+## Indexing JSON arrays with tags
```
Comment (oshadmi) [inline docs/Indexing_JSON.md:73]: Can this kindly be added also to the corresponding [page](https://oss.redis.com/redisjson/indexing_JSON/) on RedisJSON?

Comment (slorello89) [reply docs/Indexing_JSON.md:73]: See RedisJSON/RedisJSON#629 - I opened and have been maintaining them in parallel :)

**Review by oshadmi — APPROVED** [review]
👍🏼

File: docs/Tags.md [hunk 4]
```diff
@@ -114,7 +114,38 @@ For example, querying for the tag `Andrew's Top 5` in the redis-cli looks like t
...
 ```
 
+## Indexing JSON arrays with tags
 
+It is possible to index scalar string and boolean values in JSON arrays by using the wildcard operator in the JSON Path. For example if you were indexing blog posts you might have a field called `tags` which is an array of tags that apply to the blog post.
+
+```JSON
+{
+   "title":"Using RediSearch is Easy and Fun",
+   "tags":["redis","json","redissearch"]
```
Comment (K-Jo) [inline docs/Tags.md:124]: We should probably not use "tags" as a field here since it's confusing

**Pull request closed without merging**